R/RLearner_classif_boosting.R

```diff
@@ -47,3 +47,12 @@ predictLearner.classif.boosting = function(.learner, .model, .newdata, ...) {
     return(as.factor(p$class))
   }
 }
+
+#' @export
+#' @rdname getFeatureImportanceLearner
+getFeatureImportance.classif.boosting = function(.learner, .model, ...) {
+  mod = getLearnerModel(.model)
+  fiv = as.numeric(mod$importance[.model$features])
+  names(fiv) = .model$features
```

Comment:
a bit shorter in the last line: `setNames(fiv, .models$features)` return would not be needed then